web/src/features/hfiCalculator/components/PrepLevelCell.tsx

```diff
@@ -52,81 +51,38 @@ const useStyles = makeStyles({
     ...fireTableStyles.calculatedPlanningCell,
     border: '2px solid ' + prepLevelColours.bloodRed,
     background: DAILY_BACKGROUND_COLOR
-  },
-  meanPrepLevel1: {
-    ...fireTableStyles.calculatedPlanningCell,
-    background: prepLevelColours.green
-  },
-  meanPrepLevel2: {
-    ...fireTableStyles.calculatedPlanningCell,
-    background: prepLevelColours.blue
-  },
-  meanPrepLevel3: {
-    ...fireTableStyles.calculatedPlanningCell,
-    background: prepLevelColours.yellow
-  },
-  meanPrepLevel4: {
-    ...fireTableStyles.calculatedPlanningCell,
-    background: prepLevelColours.orange
-  },
-  meanPrepLevel5: {
-    ...fireTableStyles.calculatedPlanningCell,
-    background: prepLevelColours.brightRed,
-    color: 'white'
-  },
-  meanPrepLevel6: {
-    ...fireTableStyles.calculatedPlanningCell,
-    background: prepLevelColours.bloodRed,
-    color: 'white'
   }
 })
 
 const PrepLevelCell = (props: PrepLevelCellProps) => {
   const classes = useStyles()
 
-  const prepLevel = calculatePrepLevel(props.meanIntensityGroup)
+  const prepLevel = () => {
```

Comment:
Why does this have to be a function?
```suggestion
  const prepLevel =calculatePrepLevel(props.meanIntensityGroup)
```